src/marvin/core/ChatCompletion/providers/anthropic/__init__.py

```diff
@@ -121,10 +121,10 @@ def prepare_request(self, **kwargs):
         It sets the prompt to the messages in the request.
         """
         request = super().prepare_request(**kwargs)
-        request.prompt = ""
```

Comment:
From here down looks like an overeager find/replace, I think these are different from the method